Sources/Rendering/Core/VolumeMapper/SliceHelper.js

```diff
@@ -0,0 +1,82 @@
+import macro from 'vtk.js/Sources/macros';
+import * as vtkMath from 'vtk.js/Sources/Common/Core/Math';
+import vtkPlane from 'vtk.js/Sources/Common/DataModel/Plane';
+
+function vtkSliceHelper(publicAPI, model) {
+  model.classHierarchy.push('vtkSliceHelper');
+
+  model.clipPlane1 = vtkPlane.newInstance();
```

Comment:
if you do not expose it with set/get, why do you need to store it into `model` ? Does it need to be serialized ?

Reply:
I store it in case it is subclassed, but I'll prefix the names with '_' so they don't get serialized.